code/modules/materials/Mat_Materials.dm

```diff
@@ -400,6 +400,23 @@
 		setProperty("hard", 30)
 		return ..()
 
+/datum/material/metal/neutronium
+	mat_id = "neutronium"
+	name = "neutronium"
+	desc = "Neutrons condensed into a solid form."
+	color = "#043e9b"
+	material_flags = MATERIAL_ENERGY | MATERIAL_METAL
+	alpha = 255
+
+	New()
+		setProperty("density", 10)
+		setProperty("hard", 10)
+		setProperty("electrical", 70)
+		setProperty("stability", 20)
+		setProperty("n_radioactive", 85)
+		return ..()
+
```

Comment:
should maybe have some material procs a la cerenkite for irradiating people who hold it etc?

Reply:
I thought those were automatically added in Mat_Properties.dm?

Reply:
it works as is in testing, the procs were already written in another .dm

Reply:
yea all that stuff is in mat_properties.dm, same place cerenkites procs are

Reply:
line 315 of mat_properties.dm is were the n_radioactive property is handled, and also were the procs are

Reply:
👌